politeiawww/cmd/politeiawwwcli/commands/newproposal.go

```diff
@@ -13,6 +13,31 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+// Help message displayed for the command 'politeiawwwcli help newproposal'
+var NewProposalCmdHelpMsg = `newproposal "markdownFile" "attachmentFiles" 
+
+Submit a new proposal to Politeia. Proposal must be a 
+markdown file. Attachment files can be of any file type. 
+
+Arguments:
+1. markdownFile      (string, required)   Proposal 
+2. attachmentFiles   (string, optional)   Attachments 
+
+Result:
+
+{
```

Comment:
Can we use the same format here that we do with the authorize vote command?  i.e. state the variable type in parentheses followed by a short description of the variable.  All with proper text alignment.

Reply:
Good catch. Have reformatted. Pasted what I have now below in case you want further tweaks E.g. do we want to explicity specify that the variables under `files` are in a json, or is that implied by the structure?.

```
Result:

{
  "files": [
    {
      "name":      (string)  Filename 
      "mime":      (string)  Mime type 
      "digest":    (string)  File digest 
      "payload":   (string)  File payload 
    }
  ],
  "publickey":   (string)  Public key of user
  "signature":   (string)  Signed merkel root of files in proposal 
}
```